python/zensical/__init__.py

```diff
@@ -21,8 +21,8 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 # IN THE SOFTWARE.
 
-from .zensical import *
+from zensical.zensical import *  # noqa: F403
 
-__doc__ = zensical.__doc__
-if hasattr(zensical, "__all__"):
-    __all__ = zensical.__all__
+__doc__ = zensical.__doc__  # type: ignore[name-defined]  # noqa: F405
+if hasattr(zensical, "__all__"):  # type: ignore[name-defined]  # noqa: F405
+    __all__ = zensical.__all__  # type: ignore[name-defined]  # noqa: F405
```

Comment:
We could explicitly import known objects instead of using a wildcard import, to avoid the warnings (type-ignore, noqa), though that means we have to think about updating the file if the Rust module changes what it exposes.